tasks/configuration/Linux.yml

```diff
@@ -50,13 +51,14 @@
   retries: 20
   delay: 10
   when:
+    - solr_auth_configure
     - solr_auth_type is defined
```

Comment:
Should we remove it?
`solr_auth_type is defined` in defaults so it be always True